ledger-service/http-json/src/test/scala/com/digitalasset/http/Generators.scala

```diff
@@ -26,17 +26,17 @@ object Generators {
       e <- Gen.identifier
     } yield domain.TemplateId(p, m, e)
 
-  def nonEmptySet[A](gen: Gen[A]): Gen[Set[A]] = Gen.nonEmptyListOf(gen).map(_.toSet)
+  def nonEmptySetOf[A](gen: Gen[A]): Gen[Set[A]] = Gen.nonEmptyListOf(gen).map(_.toSet)
 
   // Generate Identifiers with unique packageId values, but the same moduleName and entityName.
-  def genDuplicateApiIdentifiers: Gen[List[lav1.value.Identifier]] =
+  def genDuplicateModuleEntityApiIdentifiers: Gen[Set[lav1.value.Identifier]] =
```

Comment:
This still has `duplicate` in the name but it only returns uniques now. So what is duplicate?

Reply:
The `moduleName` and `entityName`.

Reply:
Oh I see, makes sense. I should read the comment :innocent: